recipes-graphics/mesa/files/0001-dri2-query-dma-buf-modifiers.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/src/gallium/state_trackers/dri/dri2.c b/src/gallium/state_trackers/dri/dri2.c
```

Comment:
Add upstream status - see other patches for reference. https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines

Reply:
Can we also have a description of the patch in both the patch file and git log?

Reply:
- Upstream status was forgotten.
- SOB in commit log (capital letter for "signed")
- Please use your name before the email - see other commits as an example and/or use `-s` argument of git commit (and co).